src/bindgen/index.md

```diff
@@ -20,6 +20,8 @@ use Rust in a C/C++ codebase in the first place!
 
 ## Getting Set Up
 
+> **Note:** Bindgen **requires** clang. Installation instructions can be found [here](https://github.com/rust-lang-nursery/rust-bindgen/blob/master/book/src/requirements.md#installing-clang-39)
```

Comment:
I'm confused, how did adding a note lead to unformatted rust code? ![my browser view](https://user-images.githubusercontent.com/28968644/29001091-b7cf43ce-7a44-11e7-8564-67b554150a61.png)